starlette/testclient.py

```diff
@@ -398,6 +397,7 @@ class TestClient(httpx.Client):
     __test__ = False
     task: Future[None]
     portal: anyio.abc.BlockingPortal | None = None
+    redirect: bool | httpx._client.UseClientDefault = httpx._client.USE_CLIENT_DEFAULT
```

Comment:
Why this? There's a parameter in the init

Reply:
My bad, I forgot it there